### PR TITLE
Misha

### PR DESCRIPTION
**Pull request by MishaFomenko**
changed to ts:
- notification-svc;
- appointment-svc;
- appointnent-interface;
changed the above-mentioned files from commonjs to es2016

### CODE REVIEW

File: appointment-svc/models/init_app_exc_receiver.ts [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+import amqp from 'amqplib/callback_api'
+import appActionEmitter from '../utils/appointmentActionEmitter'
 require('dotenv').config();
 
 async function receiveMessageFromInitAppExc() {
 
     var router_keys = ['book', 'cancel'];
 
-    amqp.connect(process.env.RABBITMQ_URL, function (error0, connection) {
+    process.env.RABBITMQ_URL && amqp.connect(process.env.RABBITMQ_URL, function (error0, connection) {
```
Comment (vlad-lychak) [inline appointment-svc/models/init_app_exc_receiver.ts:9]: Why do you put the operator '&&' here and in other files?

Comment (MishaFomenko) [reply appointment-svc/models/init_app_exc_receiver.ts:9]: cuz ts wanted me to check for process.env.RABBITMQ_URL being potentially undefined. I guess it is called narrowing in ts

Comment (vlad-lychak) [reply appointment-svc/models/init_app_exc_receiver.ts:9]: Gotcha, also, why are you declaring a variable with 'var'?

Comment (MishaFomenko) [reply appointment-svc/models/init_app_exc_receiver.ts:9]: hz:/

Comment (MishaFomenko) [reply appointment-svc/models/init_app_exc_receiver.ts:9]: all the variables in rabbitmq tutorials defined this way for some reason. Kinda didnt want to change it not to break anything, but I actually did not notice it, so I will look into that later

**Pull request merged**